CHANGES.md

```diff
@@ -0,0 +1,6 @@
+
+# API Breaking Changes
+
+## NetworKit 5.1
```

Comment:
```suggestion
## NetworKit 6.0
```

Reply:
Actually, this should be 7.0 now, since 6.0 is already out.